core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -463,14 +461,20 @@ import scala.util.control.NonFatal
         log.error(e, "Exception when polling from consumer, stopping actor: {}", e.toString)
         context.stop(self)
     }
-    checkRebalanceState(initialRebalanceInProgress)
```

Comment:
This could possibly stay. It applies if commits arrive during a rebalance, but they'll be handled with the next poll.

Reply:
The `checkRebalanceState` method seems redundant with `commitAggregatedOffsets`.  Also, I don't see how we could have any pending commits to make anyway since the actor is currently handling the poll message.

Reply:
Yes, I came to that conclusion, as well.